src/VisualStudio/Core/Def/Implementation/LanguageClient/AbstractInProcLanguageClient.cs

```diff
@@ -85,9 +86,9 @@ internal abstract class AbstractInProcLanguageClient : ILanguageClient
         /// <summary>
         /// Can be overridden by subclasses to control what capabilities this language client has.
         /// </summary>
-        protected internal abstract VSServerCapabilities GetCapabilities();
+        protected internal abstract Task<VSServerCapabilities> GetCapabilitiesAsync(CancellationToken cancellationToken);
```

Comment:
Why make this async? Future proofing?

Reply:
i can revert.  i needed it in something experiemntal i was doing.  but didn't need it at the end